binary_search_tree/binary_search_tree.py

```diff
@@ -5,10 +5,51 @@ def __init__(self, value):
     self.right = None
 
   def insert(self, value):
-    pass
+    if self.value == value:
+      return value
+    if self.left is None:
+      if value < self.value:
+        print(f'{value} is less than {self.value}. inserting in left side of {self.value}')
```

Comment:
hoping this was able to pass the tests for you, also trying wrapping your recursive value in order to reduce redundancy